source/gent/tier1_hortense.rst

```diff
@@ -716,6 +719,39 @@ If users can provide a proper case and motivation, you can contact support to re
 to give you access to all the available resources.
 
 
+Gradual decommissioning
+-----------------------
+
+End 2025, the gradual decommissioning of Tier-1 Compute Hortense will be initiated.
+Around this time, the 4th VSC Tier-1 cluster will become available at the VUB datacenter.
+
+The entire Rome partition is end of life November 2025, and will be shut down by end 2025.
+This implies that the partitions ``cpu_rome``, ``cpu_rome_all``, ``cpu_rome_512``, ``debug_rome`` will all disappear.
+
+
+Update to RHEL9 of Milan partition
+----------------------------------
+
+To maintain operational safety, the operating system for the Milan CPU partition will be updated to a significant new version.
+Red Hat Enterprise Linux version 9 (going up from 8) will be installed near the end of 2025.
+This implies that - end 2025 - your software and/or workflow will need to be compliant with this updated OS.
+As of cutoff 2 in 2025, compatibility of your workflow/software with the new RHEL9 operating system will be a hard requirement.
+
+Please test your workflow and software as soon as possible and ensure that you are ready for this transition.
+
+To facilitate testing, we have made two small partitions to run your tests.
+
+* partition ``cpu_milan_rhel9``
+* partition ``debug_milan_rhel9``
+
+These partitions are SOLELY intended for testing your software/workflows.
+Do not run production jobs on these partitions.
+
+XXX TODO XXX
+- how to load the modules for these partitions
```

Comment:
it's the same module swap as any other partition, so in this case `module swap cluser/dodrio/cpu_milan_rhel9` or `module swap cluser/dodrio/debug_milan_rhel9` (or select similar partition from portal)